packages/wrangler/src/api/startDevWorker/LocalRuntimeController.ts

```diff
@@ -241,7 +246,10 @@ export class LocalRuntimeController extends RuntimeController {
 		// Ignored in local runtime
 	}
 
+	_torndown = false;
```

Comment:
This was confusing me for a moment as prop initialisers are normally at the top of the class definition. Is there a reason to have it down here?

Reply:
Just colocating its definition with the method that modifies it